### DISCUSSION TITLE
Automated task retrying

### DISCUSSION THREAD

**Discussion by RealOrangeOne** [RealOrangeOne/django-tasks#6] [Ideas]
When a task fails, something should happen. Usually, that'll be firing off another task to do some cleanup. In the current implementation, this will need to be manually implemented by the user. It would be nice if `django-tasks` had an automatic way of configuring this. Perhaps per-task or per-enqueue?

**Comment by omargawdat** [comment]
is there any updates for rerunning the task of handling the failed tasks as it is so important to be exist

**Reply by RealOrangeOne** [reply]
Not really - not without some manual intervention (as you've already [mentioned](https://github.com/RealOrangeOne/django-tasks/issues/146#issuecomment-2764546950))
